### ISSUE TITLE
Edit Notification Settings

### ISSUE THREAD

**Issue by rachelle-eplow** [TroyManary/EasyPlow#63]
What's the difference between Edit Notification Settings screen and Notification Settings screen if both have the same information and the ability to save and close.  See images below
![notification settings phone number population](https://cloud.githubusercontent.com/assets/13750782/9968653/a8ca3488-5e17-11e5-9b24-fa514a505daa.JPG)
![edit notification settings](https://cloud.githubusercontent.com/assets/13750782/9968659/aeb22e6e-5e17-11e5-9ff8-ebc1571e37dd.JPG)

**Comment by TroyManary** [comment]
Hi Rachelle,
Is the first display used during the Customer Signup process and the second is uses to Edit the info after an account exists?
Troy

**Comment by rachelle-eplow** [comment]
again will verify when the new release is complete

**Comment by rachelle-eplow** [comment]
Yes the first display is during the sign up process and the second is used to edit.  Problem is it doesn't retain the informaitoninput during the sign up process.

**Comment by TroyManary** [comment]
Melissa, I am not sure if this is for you or Alex.
Please ensure that the info entered during signup is saved and appears in the edit page.

**Comment by melissa-edimensionz** [comment]
@AlexEdimensionz 
Notification settings set  on sign up isn't stored on profile when selecting Edit Notification.  Think just out of sync from changes to forms (?).
